pyvista/jupyter/__init__.py

```diff
@@ -42,7 +40,7 @@ def _validate_jupyter_backend(backend):
 
     if backend in ['server', 'client', 'trame', 'html']:
         try:
-            from pyvista.trame.jupyter import show_trame
+            from pyvista.trame.jupyter import show_trame as show_trame
```

Comment:
Is it possible to use `importlib.util.find_spec` here too?

Reply:
Maybe, though unlike the other cases I'm not sure if `show_trame` is intended to be re-exported as part of the API.  The other cases were importing only to check if a module was installed.

If you don't think `show_trame` is used here as part of the API then I can probably remove it and use importlib

Reply:
OK. Let's keep as is.